### DISCUSSION TITLE
Improving Navigation Back to Fork from Comparison View

### DISCUSSION THREAD

**Discussion by excellentiam-prog** [community/community#123189] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hello GitHub Team,

I’ve noticed that navigating back to a forked repository from the comparison view can be unintuitive, especially for new users or those not familiar with GitHub’s URL structure. Currently, one has to either click on the Browser's Back Button or manually edit the URL.

Suggestion: It would be beneficial to have a more prominent button or link that says “Back to Fork” or “Return to Repository” directly on the comparison page. This would make the process more straightforward and enhance the overall user accessibility on GitHub.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @excellentiam-prog ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
